sdks/python/apache_beam/coders/coders.py

```diff
@@ -1131,7 +1131,7 @@ def __hash__(self):
     return hash((type(self), self._elem_coder))
 
 
-class IterableCoder(FastCoder):
+class ListLikeCoder(FastCoder):
```

Comment:
For the pydoc error, I'm not sure what's going on. Adding `apache_beam.coders.coders.ListLikeCoder` to ignore_identifiers in generate_pydoc.sh makes the problem go away.

Reply:
Thanks. I was mystified by this as well.